java/pom.xml

```diff
@@ -44,16 +44,6 @@
 			<artifactId>commons-collections4</artifactId>
 			<version>4.1</version>
 		</dependency>
-        <dependency>
-            <groupId>org.apache.logging.log4j</groupId>
```

Comment:
Why did we removed log4j?

Maybe is a transitive dependency of other Package (so it will be included anyway). But its seems that we should decide which version we distribute in the GXClassR. 
Maybe I'm missing something.